modules/apps/commerce/commerce-service/src/main/java/com/liferay/commerce/internal/price/CommerceProductPriceCalculationV2Impl.java

```diff
@@ -178,8 +178,12 @@ public CommerceProductPrice getCommerceProductPrice(
 
 		CommerceDiscountValue commerceDiscountValue;
 
-		BigDecimal finalPriceWithTaxAmount = getConvertedPrice(
-			cpInstanceId, finalPrice, false, commerceContext);
+		BigDecimal finalPriceWithTaxAmount = BigDecimal.ZERO;
```

Comment:
if we don't need to calculate taxes, shouldn't this value be equal to finalPrice instead of 0?